service/controller/azure/v6/resource/configmap/desired.go

```diff
@@ -18,9 +18,14 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 
 	clusterGuestConfig := azurekey.ClusterGuestConfig(customObject)
 	configMapValues := configmap.ConfigMapValues{
-		ClusterID:    key.ClusterID(clusterGuestConfig),
-		Organization: key.ClusterOrganization(clusterGuestConfig),
-		WorkerCount:  azurekey.WorkerCount(customObject),
+		ClusterID: key.ClusterID(clusterGuestConfig),
+		// Migration is disabled because Azure is already migrated.
+		IngressControllerMigrationEnabled: false,
+		// Controller Service is enabled because it is created by the chart not
+		// by k8scloudconfig.
+		IngressControllerServiceEnabled: true,
+		Organization:                    key.ClusterOrganization(clusterGuestConfig),
+		WorkerCount:                     azurekey.WorkerCount(customObject),
```

Comment:
Azure is different because its already migrated.

Reply:
When it is migrated why is the flag for the k8scc magic still enabled?

```
IngressControllerServiceEnabled: true
```

Reply:
It looks to me like single flag can be used. Both always flip at the same time.

Reply:
Good catch, thanks!

A single flag would work and be cleaner. It will need a chart change but its small.